cups/ppd-cache.c

```diff
@@ -1364,6 +1364,30 @@ _ppdCacheCreateWithPPD(ppd_file_t *ppd)	/* I - PPD file */
 
   if ((media_type = ppdFindOption(ppd, "MediaType")) != NULL)
   {
+    static const struct {
```

Comment:
Coding style - open braces should start on a new line, just like for code. Also we like to include some C-style comments (indented with 8-column tabs to column 41) to specify what each thing is.